packages/provider/src/tests/util.test.ts

```diff
@@ -63,6 +64,7 @@ describe('UTIL FUNCTIONS', async () => {
         await env
             .getDb()
             .storeScheduledTaskStatus('0x01', ScheduledTaskNames.BatchCommitment, ScheduledTaskStatus.Completed)
+        await sleep(1000)
```

Comment:
this is a hacky fix :joy:

Reply:
i ran into this issue earlier - what's the actual race condition that this fixes?

Reply:
Stuff not being available in the mongo memory server immediately, I presume. I haven't had any failures since adding this. If it crops up again then it's not the solution.